example/lib/src/data/repositories/data_users_repository.dart

```diff
@@ -4,9 +4,9 @@ import '../../domain/repositories/users_repository.dart';
 class DataUsersRepository extends UsersRepository {
   List<User> users;
   // sigleton
-  static DataUsersRepository _instance = DataUsersRepository._internal();
+  static final DataUsersRepository _instance = DataUsersRepository._internal();
   DataUsersRepository._internal() {
-    users = List<User>();
+    users = <User>[];
```

Comment:
Is there a benefit to this change?

Reply:
As defined by new lint rules in `pedantic 1.9.0`, we can omit types in this lists to improve legibility